frontend/src/general/LogoutMenu/__tests__/LogoutMenuOpener.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { beforeEach, describe, expect, it, jest } from "@jest/globals";
+
+import { IconButton } from "@material-ui/core";
+
+import { AccountCircle } from "@material-ui/icons/";
+import LogoutMenuOpener from "../LogoutMenuOpener";
+import LogoutMenu from "../LogoutMenu";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<LogoutMenuOpener/> class component", ()=>{
```

Comment:
Linter is complaining **a lot** about code styling etc. It's making it hard to review the code because every other line in this file has a ESLint warning.

Reply:
linted in 294fc18
though you can also check out the branch to run it and look at the code there, if github is only flagging linting issues